godot-macros/src/gdextension.rs

```diff
@@ -48,6 +49,19 @@ pub fn attribute_gdextension(item: venial::Item) -> ParseResult<TokenStream> {
         .or(entry_point)
         .unwrap_or_else(|| ident("gdext_rust_init"));
 
+    let wasm_binary = match wasm_binary {
+        Some(wasm_binary) => {
+            quote! {
+                {
+                    // Ensure a string-like expression is passed
+                    let wasm_binary: &str = &#wasm_binary;
+                    format!("'{}'", wasm_binary.replace("\\", "\\\\").replace("'", "\\'"))
+                }
+            }
```

Comment:
Adding a comment, because knowing myself, I'll be tempted to extract this in the future :grin: 
```suggestion
            // Replacement is not moved out of quote! to allow non-literal expressions (e.g. constants).
            quote! {
                {
                    // Ensure a string-like expression is passed.
                    let wasm_binary: &str = &#wasm_binary;
                    format!("'{}'", wasm_binary.replace("\\", "\\\\").replace("'", "\\'"))
                }
            }
```

Reply:
Furthermore, should we validate that this ends with `.wasm`?

Reply:
> Furthermore, should we validate that this ends with `.wasm`?

Yep, added it.